code42_connector.py

```diff
@@ -119,6 +130,20 @@ def _handle_list_departing_employees(self, param, action_result):
         action_result.update_summary({"total_count": total_count})
         return action_result.set_status(phantom.APP_SUCCESS)
 
+    @action_handler_for("get_departing_employee")
+    def _handle_get_departing_employee(self, param, action_result):
+        username = param["username"]
+        user_id = self._get_user_id(username)
+
+        try:
+            response = self._client.detectionlists.departing_employee.get(user_id)
+            action_result.add_data(response.data)
+            action_result.update_summary({"is_departing_employee": True})
```

Comment:
The reason these `get` actions were requests in XSOAR is because they wanted to be able to check if a user was in fact a certain detection list occupant.

So, the summary tells them that and even provided a bool to trip off of.

If they are a list occupant, they will also get the data of the API response, such as the `userId` etc